api/TimeAddressableMediaStore.yaml

```diff
@@ -28,7 +28,7 @@ info:
     could for example be implemented using an AWS S3 compatible store using presigned URLs or by
     a simple file system storage with an HTTP frontend. Clients should parse the /service endpoint
     and handle the media store type in use appropriately.
-  version: "2.0"
+  version: "2.1"
```

Comment:
Remove this version change now that the release tooling has been added?

Reply:
Good point, removed